CHANGELOG.md

```diff
@@ -1,12 +1,149 @@
-* Add `alloc_str`, which is similar to `alloc_slice_*` methods, but works on
-  string slices.
+## Unreleased
+
+Released YYYY-MM-DD.
+
+### Added
+
+* TODO (or remove section if none)
+
+### Changed
+
+* TODO (or remove section if none)
+
+### Deprecated
+
+* TODO (or remove section if none)
+
+### Removed
+
+* TODO (or remove section if none)
+
+### Fixed
+
+* TODO (or remove section if none)
+
+### Security
+
+* TODO (or remove section if none)
+
+--------------------------------------------------------------------------------
+
+# 3.0.0
+
+Released 2019-12-20.
+
+## Added
+
+* Added `Bump::alloc_str` for copying string slices into a `Bump`.
+
+* Added `Bump::alloc_slice_copy` and `Bump::alloc_slice_clone` for copying or
+  cloning slices into a `Bump`.
+
+* Added `Bump::alloc_slice_fill_iter` for allocating a slice in the `Bump` from
+  an iterator.
+
+* Added `Bump::alloc_slice_fill_copy` and `Bump::alloc_slice_fill_clone` for
+  creating slices of length `n` that are filled with copies or clones of an
+  inital element.
+
+* Added `Bump::alloc_slice_fill_default` for creating slices of length `n` with
+  the element type's default instance.
+
+* Added `Bump::alloc_slice_fill_with` for creating slices of length `n` whose
+  elements are initialized with a function or closure.
+
+* Added `Bump::iter_allocated_chunks` as a replacement for the old
+  `Bump::each_allocated_chunk`. The `iter_allocated_chunks` version returns an
+  iterator, which is more idiomatic than its old, callback-taking counterpart.
+  Additionally, `iter_allocated_chunks` exposes the chunks as `MaybeUninit`s
+  instead of slices, which makes it usable in more situations without triggering
+  undefined behavior. See also the note about bump direction in the "changed"
+  section; if you're iterating chunks, you're likely affected by that change!
+
+* Added `Bump::with_capacity` so that you can pre-allocate a chunk with the
+  requested space.
+
+### Changed
+
+* **BREAKING:** The direction we allocate within a chunk has changed. It used to
+  be "upwards", from low addresses within a chunk towards high addresses. It is
+  now "downwards", from high addresses towards lower addresses.
+
+  Additionally, the order in which we iterate over allocated chunks has changed!
+  We used to iterate over chunks from oldest chunk to newest chunk, and now we
+  do the opposite: the toungest chunks are iterated over first, and the oldest
+  chunks are iterated over last.
+
+  If you were using `Bump::each_allocated_chunk` to iterate over data that you
+  had previously allocated, and *you want to iterate in order of allocation*,
+  you need to reverse the chunks iterator and also reverse the order in which
+  you loop through the data within a chunk!
```

Comment:
Have you considered also writing something about the maximum supported alignment size? I am not sure if it actually matters enough though.